api/simplyrets-swagger.yaml

```diff
@@ -126,6 +126,537 @@ info:
 
         Using this request, look for the `lastUpdate` field in the JSON
         response.
+components:
```

Comment:
swagger v2 uses a different type of construct here, not components:

![image](https://user-images.githubusercontent.com/526560/83285941-bfa21380-a1a4-11ea-8d5e-debe9fb0c194.png)

It's not technically wrong but wouldn't be usable as an import into assertible

Reply:
@creichert - yeah, fixed